### PR TITLE
[RUN-4153] Update System requirements prerequisites

### PR DESCRIPTION
**Pull request by akkrishnakumar**

### CODE REVIEW

File: content/en/docs/refguide/installation/system-requirements.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ The following frameworks are required. They will be installed automatically by t
     
 * Microsoft Visual C++ 2019 Redistributable Package (x64)
 * A Java Developer Kit (JDK) version 11, 17, or 21 - if not yet installed on your machine, Mendix will install 'Eclipse Temurin JDK 21 (x64)'
+* Gradle 7.6 or above.
```
Comment (atennapel) [inline content/en/docs/refguide/installation/system-requirements.md]: For the JDK it mentions " - if not yet installed on your machine, Mendix will install 'Eclipse Temurin JDK 21 (x64)'". Maybe for Gradle we can mention that it installs Gradle 8.5.

**Comment by atennapel** [comment]
Looks good to me.

File: content/en/docs/refguide/installation/system-requirements.md (hunk 1 above)
Comment (atennapel) [reply content/en/docs/refguide/installation/system-requirements.md]: Looks good. (I cannot resolve this comment)

Comment (akkrishnakumar) [reply content/en/docs/refguide/installation/system-requirements.md]: Its ok. Mark will resolve it

Comment (MarkvanMents) [reply content/en/docs/refguide/installation/system-requirements.md]: Resolving

File: content/en/docs/refguide/installation/system-requirements.md [hunk 2]
```diff
@@ -47,6 +47,7 @@ The following frameworks are required. They will be installed automatically by t
     
 * Microsoft Visual C++ 2019 Redistributable Package (x64)
 * A Java Developer Kit (JDK) version 11, 17, or 21 - if not yet installed on your machine, Mendix will install 'Eclipse Temurin JDK 21 (x64)'
+* Gradle version 7.6 or above - if Gradle is not yet installed on your machine, Mendix will install Gradle version 8.5
```
Comment (MarkvanMents) [inline content/en/docs/refguide/installation/system-requirements.md]: Was this introduced in a specific Mendix version? If so, which?

File: content/en/docs/refguide9/general/system-requirements.md [hunk 3]
```diff
@@ -28,6 +28,7 @@ The following frameworks are required. They will be installed automatically by t
     * Adoptium Temurin JDK 11 (x64) – for Mendix 9.14 to 9.17
     * Eclipse Temurin JDK 11 (x64)– for Mendix between 9.18.1 and 9.18.15
     * Eclipse Temurin JDK 17 (x64)– for Mendix 9.18.16 and above
+* Gradle version 7.6 or above - if Gradle is not yet installed on your machine, Mendix will install Gradle version 8.5
```
Comment (MarkvanMents) [inline content/en/docs/refguide9/general/system-requirements.md]: Was this introduced in a specific Mendix version? If so, which?

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
Hi @akkrishnakumar and @atennapel 
Thanks for the PR.
I'm afraid I'm off on holiday now. @dbreseman could you follow up on this last question and update the documentation accordingly (see JDK documentation just above this change) if these do not apply to all Mx9/10 versions. Or pass it on to @MariaShaposhnikova if you need someone else to deal with it.

Also, can this be published immediately, or is it related to 10.13/9.24.26?

File: content/en/docs/refguide9/general/system-requirements.md (hunk 3 above)
Comment (atennapel) [reply content/en/docs/refguide9/general/system-requirements.md]: Gradle 7.6 was added in 9.24.0. Updated to 7.6.3 in 10.4.0 and updated to 8.5 in 10.10.0.

File: content/en/docs/refguide/installation/system-requirements.md (hunk 2 above)
Comment (atennapel) [reply content/en/docs/refguide/installation/system-requirements.md]: Gradle 7.6 was added in 9.24.0. Updated to 7.6.3 in 10.4.0 and updated to 8.5 in 10.10.0.

**Comment by atennapel** [comment]
Looks good to me.

File: content/en/docs/refguide/installation/system-requirements.md (hunk 2 above)
Comment (dbreseman) [reply content/en/docs/refguide/installation/system-requirements.md]: Hi @atennapel, does this mean that Gradle 8.5 or higher is required for SP 10.0.0 onwards?

Comment (atennapel) [reply content/en/docs/refguide/installation/system-requirements.md]: Gradle 8.5 is required IF you set your application Java version to 21. If your Java version is 11 or 17 then Gradle 7.6/7.6.3 will also work. Note that in Studio Pro you will also get a consistency error if you use Gradle 7.6.x and Java 21.

**Pull request merged**